packages/core-typings/src/ILivechatContact.ts

```diff
@@ -37,4 +40,5 @@ export interface ILivechatContact extends IRocketChatRecord {
 		ts: Date;
 	};
 	importIds?: string[];
+	activity?: YearMonthString[];
```

Comment:
hmmmm... not sure this provides much value 🤔 can't we keep with just `string` and the comment above here?